src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,102 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static final String DEFAULT_LOGIN = "Karabas";
+    private static final String DEFAULT_PASSWORD = "BARABAS";
+    private static final int DEFAULT_AGE = 20;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_userIsNull_notOk() {
+        assertThrows(ValidationException.class, () -> {
+            registrationService.register(null);
+        });
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        assertThrows(ValidationException.class, () -> {
+            registrationService.register(new User(null, "DEFAULT_PASSWORD", DEFAULT_AGE));
```

Comment:
```suggestion
            registrationService.register(new User(null, DEFAULT_PASSWORD, DEFAULT_AGE));
```
please fix similar issue in all places